examples/deepcopy-gen/generators/deepcopy.go

```diff
@@ -719,6 +758,8 @@ func (g *genDeepCopy) doSlice(t *types.Type, sw *generator.SnippetWriter) {
 			sw.Do("}\n", nil)
 		} else if t.Elem.Kind == types.Struct {
 			sw.Do("(*in)[i].DeepCopyInto(&(*out)[i])\n", nil)
+		} else if t.Elem.Kind == types.Alias {
```

Comment:
Factored out the alias change and moved it to https://github.com/kubernetes/gengo/pull/73. That's independent and nice to have.